libs/model/src/test/seed.ts

```diff
@@ -14,68 +17,96 @@ const GENERATED_PROPS = [
 ] as const;
 type GeneratedProp = typeof GENERATED_PROPS[number];
 
-// incrementing seed number for each call to the `seed` function
-// for deterministic randomness when the order of calls is preserved
-let seedNum = 1;
-
-//type E = keyof typeof schemas;
-
-export type SchemaWithModel<T extends z.AnyZodObject> = {
-  schema: T;
-  model: ModelStatic<Model>;
-  mockDefaults?: () => Partial<z.infer<T>>;
-  allowedGeneratedProps?: GeneratedProp[];
-  buildQuery?: (data: z.infer<T>) => {
-    where: Partial<z.infer<T>>;
-  };
-};
-
+/**
+ * Seed options
+ *
+ * @param mock true to auto mock values in schemas
+ * @param skip array of field names to remove from schemas when auto mocked
+ * @param log log new records to console
+ */
 export type SeedOptions = {
   mock: boolean;
+  skip?: GeneratedProp[];
+  log?: boolean;
 };
 
-export async function seed<T extends SchemaWithModel<any>>(
-  { schema, model, mockDefaults, allowedGeneratedProps }: T,
-  overrides: Partial<z.infer<T['schema']>> = {},
+/**
+ * Seeds aggregates for unit testing
+ *
+ * @param name the name of the entity
```

Comment:
```suggestion
 * @param name of the entity
```

Reply:
I usually repeat the param name in the description... curious if you have found a good reference for naming in jsdoc?

Reply:
Oh woops my bad I read this as a typo your correct here we should revert this. I believe the standard allows use to use a hyphen between the parameter name and the description which would make this a little more readable. Maybe we should start doing that? Probably something to discuss with @gdjohnson for internal TSDoc docs.